src/main/java/com/epam/mentoring/homeWork/module5/task2/DisplayInfoFromPerson.java

```diff
@@ -9,15 +12,62 @@
 
 
 public class DisplayInfoFromPerson {
+    private static final Logger LOGGER = LoggerFactory.getLogger(DisplayInfoFromPerson.class);
+
     public static void main(String[] args) {
         HashMap<Passport, Person> personInfo = new HashMap<Passport, Person>();
```

Comment:
Code convention. Use interface rather then concrete type `Map<Passport, Person> personInfo = new HashMap<>();`
If you ever want to be able to operate on a different object type in the future, it’s much more straightforward to make it implement an interface rather than try to force it into an existing class hierarchy